### DISCUSSION TITLE
I have created a file that supports Korean.

### DISCUSSION THREAD

**Discussion by OVJECT** [GetPublii/Publii#1483] [General]
I have created a file that supports Korean.
Is there any way I can share it?

**Comment by OVJECT** [comment]
https://github.com/OVJECT/Publii-ui-locales
I'm not good at using GitHub
I put it here first
hopefully it will be uploaded to the official GitHub!

**Reply by dziudek** [reply]
@OVJECT - I has been able to merge your work using pull request. Thank you for your contribution :)
